packages/backend/src/open_payments/payment_pointer/key/routes.ts

```diff
@@ -1,6 +1,6 @@
 import { generateJwk, JWK } from 'http-signature-utils'
 
-import { PaymentPointerContext } from '../../../app'
+import { PaymentPointerContext, PaymentPointerKeysContext } from '../../../app'
```

Comment:
```suggestion
import { PaymentPointerKeysContext } from '../../../app'
```
Make everything `PaymentPointerKeysContext`